examples/generative/corrdiff/train.py

```diff
@@ -84,6 +85,7 @@ def main(cfg: DictConfig) -> None:
     wandb_mode = getattr(cfg, "wandb_mode", "disabled")
     tick = getattr(cfg, "tick", 1)
     dump = getattr(cfg, "dump", 500)
+    validation_dump = getattr(cfg, "validation_dump", 500)
```

Comment:
Can we make this configurable by adding a `validation_dump` item to  `config_train_base.yaml`?